css-responsive/css-responsive_playground/css/styles.css

```diff
@@ -94,3 +93,34 @@ h1 {
 .box--five img {
   width: 100%;
 }
+
+@media (min-width: 599px) {
```

Comment:
I think when the instructions are greater than X we should do x+1 so 600 in this case